docs/pages/play.vue

```diff
@@ -160,29 +160,46 @@
 <script>
 import Vue from 'vue'
 import debounce from 'lodash/debounce'
+import { transform, disableScriptTags } from '@babel/standalone'
```

Comment:
I would suggest lazy importing this script on mounted and meanwhile show a progress indicator.

Reply:
That is a good idea...  I was thinking about async require

Reply:
For now, I'm just going to try it as regular import to make sure compilation and error trapping work properly.